packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -496,6 +526,15 @@ export async function copyProjectTemplateAndReplace(
   }
 }
 
+function getUwpCsPackages(): NugetPackage[] {
+  return [
+    {
+      id: 'Microsoft.NETCore.UniversalWindowsPlatform',
+      version: '6.2.9',
```

Comment:
unrelated but we probably want to upgrade this number now

Reply:
I'm guessing we'd want to do this in a separate issue - filed #9194.